local-notifications/src/definitions.ts

```diff
@@ -163,6 +163,42 @@ export interface LocalNotificationDescriptor {
    * @since 1.0.0
    */
   id: string;
+
+  /**
+   * The title of the notification.
+   *
+   * @since 1.0.0
+   */
+  title?: string;
```

Comment:
`title` isn't optional when creating notifications, I don't think it can be undefined